apps/cms/models.py

```diff
@@ -49,6 +49,12 @@ def value_for_form(self, value):
             return value.pk
         return value
 
+    def value_from_form(self, value):
+        # if project became unavailable (unpublished), selection will become an
+        # empty string and cause a server error on save, so we give a fallback
+        value = value or None
+        return super().value_from_form(value)
```

Comment:
This is a wagtail bug. We should add a note that this can be removed once the bug is fixed.